### ISSUE TITLE
[Enhancement] More useful issues search link in the error message page

### ISSUE THREAD

**Issue by opk12** [iv-org/invidious#4527]
**Is your enhancement request related to a problem? Please describe.**
People don't search before opening a duplicate, even if they are given the link `searched for existing issues on GitHub` which points to

    https://github.com/iv-org/invidious/issues

**Describe the solution you'd like**
The `searched for existing issues on GitHub` link should be prefilled with the exception class; or with the message, if it's the same for all users. For example, for #4498,

    https://github.com/iv-org/invidious/issues?q=is%3Aissue+is%3Aopen+VideoNotAvailableException

**Comment by unixfox** [comment]
The solution is to force a github issue template. There is a way to force to check a checkbox, this will 100℅ help reducing the duplicated issues.

**Comment by opk12** [comment]
I disagree that an army of newbies can be "forced", but putting that aside, changing the URL would help anyone who clicks on the link.

**Comment by SamantazFox** [comment]
I agree with @opk12 here. Pre-filling the search bar is a good idea.
The template has high chances to be ignored.

**Comment by ChunkyProgrammer** [comment]
With issue forms, the template cant be easily ignored (you can mark some fields as required) 
Here's an example of how it looks (this is what's used for freetube bug reports)
![Screenshots_2024-03-30-14-27-22](https://github.com/iv-org/invidious/assets/78101139/66293686-adf4-4d3a-ae5e-6051aaaa64c6)
